tinker/templates/events/home.html

```diff
@@ -22,10 +22,10 @@
                     {% endif %}
                     <br/>
                     {% if form.is_published %}
-{#                                            <small>#}
+                        {#                                            <small>#}
```

Comment:
you can probably remove these commented out lines

Reply:
Done